site/gdocs/OwidGdocPage.tsx

```diff
@@ -29,6 +32,14 @@ export default function OwidGdocPage({
 }) {
     const { content, slug, createdAt, updatedAt } = gdoc
 
+    const featuredImageUrl =
+        content["featured-image"] &&
+        path.join(
+            baseUrl,
+            "images",
+            "published",
```

Comment:
`images/published` is being used in a few places by now, it might be worth saving that path into a variable